toolkit/scripts/toolchain/container/toolchain_build_in_chroot.sh

```diff
@@ -57,14 +57,14 @@ set -e
 #
 cd /sources
 
-echo Linux-5.10.117.1 API Headers
-tar xf kernel-5.10.117.1.tar.gz
-pushd CBL-Mariner-Linux-Kernel-rolling-lts-mariner-5.10.117.1
+echo Linux-5.10.123.1 API Headers
```

Comment:
There is a streamlining change Neha recently made in 2.0 that you could apply here 

https://github.com/microsoft/CBL-Mariner/pull/3188/files#diff-e0a5d8c1299d7f9041d8616065b1b8e2fc75b27723495322c6b2281a8f230ebeL117